mindee/parsing/common/product.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional
+
+class Product:
+    """Class for keeping track of a product's info."""
+    name: Optional[str]
+    version: Optional[str]
```

Comment:
these will always be set
```suggestion
    name: str
    version: str
```